spec.md

```diff
@@ -496,13 +496,17 @@ When using the `last` query parameter, the `n` parameter is OPTIONAL.
 Clients should see [client-implementation.md](client-implementation.md) for more details on implementing this.
 Registries should see [upgrading.md](upgrading.md) before enabling this.*
 
-To fetch the list of referrers, perform a `GET` request to a path in the following format: `/v2/<name>/referrers/<reference>` <sup>[end-12a](#endpoints)</sup>
+To fetch the list of referrers, perform a `GET` request to a path in the following format: `/v2/<name>/referrers/<reference>` <sup>[end-12a](#endpoints)</sup>.
 
 `<name>` is the namespace of the repository.
 Assuming a repository is found, this request MUST return a `200 OK` response code.
-If a query results in no referrers found, an empty manifest list MUST be returned.
 
-Upon success, the response MUST be a json body in the following format (an index / manifest list):
+Upon success, the response MUST be a json body with an Index containing a list of descriptors.
+Each descriptor is of an Image or Artifact manifest in the same `<name>` namespace with a `refers` field that specifies the value of `<reference>`.
+The descriptors MUST include an `artifactType` field that is set to the value of `artifactType` for an Artifact manifest, or the configuration descriptor's `mediaType` for an Image manifest.
```

Comment:
I think it is optional to include the `artifactType` in the artifact manifest now, so maybe this should say:
```suggestion
The descriptors MUST include an `artifactType` field that is set to the value of `artifactType` for an Artifact manifest if present, or the configuration descriptor's `mediaType` for an Image manifest.
```

Reply:
Done, thanks!